extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -842,7 +843,7 @@ descendant_tag_list: {
   tag: "VKERN"
   tag: "WBR"
 }
-tags: {  # <amp-story-page-attachment> with href
+tags: {  # <amp-story-page-attachment> with href. Legacy version of <amp-story-page-outlink>
```

Comment:
Please line wrap at 80 columns. You may want to put the comment on the previous line.